tests/api/test_views.py

```diff
@@ -66,7 +68,7 @@ def create_items(self, factory, items):
                     'email': 'test1@example.com',
                     'is_staff': True,
                     'is_active': False,
-                    'date_joined': (now - datetime.timedelta(days=10)).isoformat(),
+                    'date_joined': (now - datetime.timedelta(days=10)).strftime(LMS_API_DATETIME_FORMAT),
```

Comment:
[See above](https://github.com/edx/edx-enterprise/pull/69/commits/835e06a373651c57c2fc0165f4e3287ac20efaae#r106059492) for why this change and similar below were required.